templates/dataset.yaml

```diff
@@ -20,19 +20,55 @@ dataset:
     # where applicable. By the default, it is assumed that your custom dataset uses EFont's variable naming
     # convention. Any other variables in your dataset that cannot be mapped to the EFont variables will be
     # ignored and cannot be used for calculations at this time.
-    variable_name:
+     timestamps: timestamps
+     baseline_electric_power_profile: 
+        variable_name:
+            name: variable_name 
         building_id:
-            name: building_id
-        availability:
-            name: availability
-            filepath: filepath/to/variable_name # [OPTIONAL] Supply filepath to dataset where variable can be found if different from <dataset>:<filepath> value
-        timestamps: timestamps
-        baseline_electric_power_profile: baseline_electric_power_profile
-        baseline_electricity_consumption_profile: baseline_electricity_consumption_profile
-        baseline_natural_gas_consumption_profile: baseline_natural_gas_consumption_profile
-        flexible_electric_power_profile: flexible_electric_power_profile
-        flexible_electricity_consumption_profile: flexible_electricity_consumption_profile
-        flexible_natural_gas_consumption_profile: flexible_natural_gas_consumption_profile
-        load_profile_peak_timestamp: load_profile_peak_timestamp
-        load_profile_valley_timestamp: load_profile_valley_timestamp
-...
\ No newline at end of file
+            name: building_id 
+        filepath: filepath/to/variable_name # [OPTIONAL] Supply filepath to dataset where variable can be found if different from <dataset>:<filepath> value
+     baseline_electricity_consumption_profile: 
```

Comment:
If I understand correctly, the structure for defining a dataset field is:

```yaml
<efont_field_name>:
    variable_name:
        name: <name>
    building_id:
        name: <building_id>
    filepath: <filepath>
```

I think we can flatten it a bit to this instead:

```yaml
<efont_field_name>:
    name: <name>
    building_id: <building_id>
    filepath: <filepath>
```

Also, does this mean that it will be one config file per building given the` <efont_field>:<building_id>` key? If that is the case then the `building_id` needs to be defined just once before mapping the field names.

If it is one config per building, is it then likely that fields for the same building will have their data in different files to need a `<efont_field>:<filepath>` key for each field?

Reply:
I like the flattened version! About the building_id, let's discuss that today. Of the 16 datasets we have, 11 have multiple buildings (some with more than 100, even 400 buildings). For some, we have different files per building with different variables in the columns; for others, we have different files per variable and then different columns per building. We should come up with a way to address both cases. Also, we should think if the intention is to compare all the buildings together with a baseline or between themselves, or if we want to compare each building with the baseline, and also what makes them suitable for KPIs for as clusters of buildings.